zerver/test_external.py

```diff
@@ -224,4 +264,3 @@ def test_change_user(self):
         self.login("hamlet@zulip.com")
         result = self.client.post('/json/users/me/android_gcm_reg_id', {'token':token})
         self.assert_json_success(result)
-
```

Comment:
Interesting, what text editor are you using?

I think the issue here is that you replaced a blank link with a line with some spaces on it; if you delete the spaces the problem and then save, the delta should go away.